src/scripts/main.js

```diff
@@ -1,3 +1,257 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+const table = document.querySelector('table');
+const header = document.querySelector('thead tr');
+const tbody = document.querySelector('tbody');
+const form = document.createElement('form');
+let isSortingAscDesc = false;
+let selectedIndexColumnForSort = -1;
+const activeTrRow = [];
+
+const formFieldsSettings = [
+  {
+    lab: 'Name:', dSet: 'name', elem: 'input', type: 'text',
+  },
```

Comment:
let's pay attention to naming here. Do not use short names such as `lab` instead of `label` and it is not clear at all what `sSet` stands for) please use more descriptive names